.github/workflows/nodejs.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [12.x]
+        node-version: [14.x]
```

Comment:
Don't change it, we use `12`, i.e. minimum LTS for linting

Reply:
OK, the intent wasn't clear there - it was running in 14 up till now, though. Updated: